plugins/woocommerce/client/admin/client/order-attribution-install-banner/order-attribution-install-banner.js

```diff
@@ -0,0 +1,174 @@
+/**
+ * External dependencies
+ */
+import { Button, Card, CardBody } from '@wordpress/components';
+import { useEffect, useCallback } from '@wordpress/element';
+import { plugins, external } from '@wordpress/icons';
+import { __, sprintf } from '@wordpress/i18n';
+import { Text } from '@woocommerce/experimental';
+import { recordEvent } from '@woocommerce/tracks';
+import { getPath } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import OrderAttributionInstallBannerImage from './order-attribution-install-banner-image';
+import { useOrderAttributionInstallBanner } from './use-order-attribution-install-banner';
+import './style.scss';
+
+const WC_ANALYTICS_PRODUCT_URL =
+	'https://woocommerce.com/products/woocommerce-analytics';
+
+export const OrderAttributionInstallBanner = ( {
+	bannerImage = <OrderAttributionInstallBannerImage />,
+	eventContext = 'analytics-overview',
+	isHeaderBanner = false,
+	isSmallBanner = false,
+	showBadge = true,
+	dismissable = true,
+	title = 'Discover what drives your sales',
+	description = 'Understand what truly drives revenue with our powerful order attribution extension. Use it to track your sales journey, identify your most effective marketing channels, and optimize your sales strategy.',
+	buttonText = 'Try it now',
+} ) => {
+	const { isDismissed, dismiss, shouldShowBanner } =
+		useOrderAttributionInstallBanner();
+
+	const onButtonClick = () => {
+		recordEvent( 'order_attribution_install_banner_clicked', {
+			path: getPath(),
+			context: eventContext,
+		} );
+	};
+
+	const getShouldRender = useCallback( () => {
+		if ( isHeaderBanner ) {
+			return shouldShowBanner && isDismissed;
+		}
+
+		if ( ! dismissable ) {
+			return shouldShowBanner;
+		}
+
+		return shouldShowBanner && ! isDismissed;
+	}, [ isHeaderBanner, shouldShowBanner, isDismissed, dismissable ] );
+
+	const shouldRender = getShouldRender();
+
+	useEffect( () => {
+		if ( ! shouldRender ) {
+			return;
+		}
+		recordEvent( 'order_attribution_install_banner_viewed', {
+			path: getPath(),
+			context: eventContext,
+		} );
+	}, [ eventContext, shouldRender ] );
+
+	if ( ! shouldRender ) {
+		return null;
+	}
+
+	if ( isHeaderBanner ) {
+		return (
+			<Button
+				className="woocommerce-order-attribution-install-header-banner"
+				href={ WC_ANALYTICS_PRODUCT_URL }
+				variant="secondary"
+				icon={ plugins }
+				size="default"
+				onClick={ onButtonClick }
+			>
+				{ __( 'Try Order Attribution', 'woocommerce' ) }
+			</Button>
+		);
+	}
+
+	if ( title ) {
+		// translators: %s: The dynamic title.
+		title = sprintf( __( '%s ', 'woocommerce' ), title );
```

Comment:
I wasn't really sure if this is a bit hacky. If I remove the space after `%s` I would get the eslint error `Translatable strings should not contain nothing but placeholders`. But I'm not sure if there's a better way to use dynamic content here.

Reply:
What about having a data file with the contents?

I would replace `isSmallBanner` & `isHeaderBanner` with a type like `bannerType=BANNER_TYPE_SMALL`, `bannerType=BANNER_TYPE_BIG`, `bannerType=BANNER_TYPE_HEADER`  

In the file we can do something like this:

```js
const BANNER_TYPE_BIG = {
'description':  __( 'Understand what truly drives revenue with our powerful order attribution extension. Use it to track your sales journey, identify your most effective marketing channels, and optimize your sales strategy.', 'woocommerce' ),
'title': __( 'Discover what drives your sales', 'woocommerce' ),
'buttonText': __('Try it now', 'woocommerce'),
}
```

Reply:
Other option may be to set the translations in the consumer side:

```jsx
OrderAttributionInstallBanner
				eventContext="order-editor-order-attribution-metabox"
				showBadge={ false }
				dismissable={ false }
				isSmallBanner
				description={ __("View all of your orders in our new Order Attribution extension.", 'woocommerce' ) }
				buttonText={ __("Install the extension", "woocommerce" ) }
			/>
```